packages/mobile/e2e/src/usecases/ExchangeCelo.js

```diff
@@ -53,30 +54,26 @@ export default ExchangeCelo = () => {
     await element(by.id('ConfirmExchange')).tap()
     // Enter PIN
     await enterPinUiIfNecessary()
+    // Get transaction time
+    let transactionTime = new Date()
     // Scroll to top of feed
     await element(by.id('ExchangeScrollView')).scroll(200, 'down')
     // Wait 10 seconds checking that error banner is not visible each second
     await waitForExpectNotVisible('errorBanner')
-    // Assert balance is updated
-    await waitFor(element(by.id('CeloBalance')))
-      .toHaveText(`${(+celoBalanceBefore.text + CELO_TO_BUY).toFixed(3)}`)
-      .withTimeout(10 * 1000)
-    // Assert buy amount is present - displays amount after fees
-    await expect(
-      element(
-        by.text(`${(CELO_TO_BUY - FEES).toFixed(3)}`).withAncestor(by.id('TransactionList'))
-      ).atIndex(0)
-    ).toBeVisible()
+    // Wait up to 1 minute and assert the transaction correctly appears
+    // TODO: assert on amount minus fees
+    await waitFor(element(by.text(`${format(transactionTime, "MMM d 'at' h':'mm a")}`)).atIndex(0))
```

Comment:
In the minute changes just as the transaction is confirmed this will fail, right?

I agree it's probably better than it used to but I'd love to avoid this sources of flakiness in the future :/ Having tests fail when they shouldn't makes developers waiting to merge their PRs waste a lot of time